client/src/components/reviews/AddReview.jsx

```diff
@@ -5,177 +5,233 @@ class AddReview extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
-      helpful: ''
+      helpful: '',
+      characteristics: {},
+      body: '',
+      summary: '',
+      name: '',
+      email: '',
+      rating: 0
     };
+    // this.description = this.description.bind(this);
   }
 
   setDescription (description) {
     if (description === 'Size') {
-      return (<div className='reviewRadio'>
+      return (<div onChange={this.description.bind(this)} className='reviewRadio'>
```

Comment:
Is it cleaner to bind this function in the constructor (like you did on line 16) since you retype it a few times?